src/Livewire/FolderTree.php

```diff
@@ -21,12 +23,12 @@ class FolderTree extends Component
 {
     use Actions, WithFileUploads;
 
-    /** @var Model $this->modelType */
+    /** @var Model $this ->modelType */
     public ?string $modelType = null;
 
     public ?int $modelId = null;
 
-    public $files = [];
+    public array $files = [];
```

Comment:
are you sure this is always an array? I remember darkly in the back of my mind, that it caused issues setting property type to array.

Reply:
will revert